{% set name = "ftfy" %}
{% set version = "5.3.0" %}
{% set sha256 = "0ba702d5138f9b35df32b55920c9466208608108f1f3d5de1a68c17e3d68cb7f" %}

package:
  name: {{ name }}
  version: {{ version }}

source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: {{ sha256 }}

build:
  number: 0
  script: python setup.py install --single-version-externally-managed --record=record.txt
  skip: True  # [py2k]
  entry_points:
    - ftfy = ftfy.cli:main

requirements:
  host:
    - python
    - setuptools

  run:
    - python
    # Required for loading package data via `pkg_resources` module.
    - setuptools
    - wcwidth

test:
  imports:
    - ftfy
    - ftfy.bad_codecs
  requires:
    - nose
  source_files:
    - tests
  commands:
    - nosetests -v tests  # [unix]
    # CLI tests fail on windows.
    - nosetests -v -e test_cli tests  # [win]
    - ftfy --help

about:
  home: https://github.com/LuminosoInsight/python-ftfy/
  license: MIT
  license_file: LICENSE.txt
  summary: Fixes some problems with Unicode text after the fact
  dev_url: https://github.com/LuminosoInsight/python-ftfy
  doc_url: http://ftfy.readthedocs.io/

extra:
  recipe-maintainers:
    - rmax
